plugins/sidebar/overlays/download.js

```diff
@@ -8,102 +8,32 @@ const Button = require('react-material/components/Button');
 const Loader = require('react-loader');
 const Progress = require('./progress');
 
-const styles = require('../styles');
-
-class DownloadOverlay extends React.Component {
-  constructor(){
-    this.state = {
-      devicePath: null,
-      searching: false,
-      selectedDevice: null,
-      devices: [],
-      progress: 0
-    };
-
-    this.onAccept = this.onAccept.bind(this);
-    this.onCancel = this.onCancel.bind(this);
-    this.reloadDevices = this.reloadDevices.bind(this);
-    this.updateSelected = this.updateSelected.bind(this);
-    this.updateProgress = this.updateProgress.bind(this);
-  }
-
-  componentDidMount(){
-    this.reloadDevices();
-  }
-
-  onAccept(evt){
-    this.download(this.state.selectedDevice);
-  }
-
-  onCancel(evt){
-    if(typeof this.props.onCancel === 'function'){
-      this.props.onCancel(evt);
-    }
-  }
-
-  download(device){
-
-    const { irken, handleError, handleSuccess } = this.props;
-
-    const { toast, workspace, logger, overlay } = irken;
-
-    const name = workspace.filename.deref();
-    const source = workspace.current.deref();
-
-    if(!device){
-      return;
-    }
+const DeviceActions = require('../../../src/actions/DeviceActions.js');
+const DeviceStore = require('../../../src/stores/DeviceStore.js');
 
-    const board = irken.getBoard(device);
+const { createContainer } = require('sovereign');
```

Comment:
all npm dependencies should be grouped together at the top (before relative dependencies)